src/Scenario/ClientGenerator.php

```diff
@@ -22,13 +22,8 @@ public function getStatusWatcherClient(StatusWatcherCallbacks $callbacks)
         return new StatusWatcherClient($callbacks);
     }
 
-    public function getAuthKeyInfo(): string
+    public function getAuthKey(): string
     {
         return trim(file_get_contents('./first.authkey'));
```

Comment:
It is better to iliminate default values - let user pass it madatory

Reply:
passed via env var now, required.